extensions/amp-tiktok/0.1/amp-tiktok.js

```diff
@@ -86,6 +87,25 @@ export class AmpTiktok extends AMP.BaseElement {
     );
   }
 
+  /** @override @nocollapse */
+  static createLoaderLogoCallback(element) {
+    const html = htmlFor(element);
+    return {
+      color: '#000000',
```

Comment:
This will make the loading circle white
```suggestion
      color: '#FFFFFF',
```